public/app/core/directives/autofill_event_fix.ts

```diff
@@ -0,0 +1,35 @@
+﻿import coreModule from '../core_module';
+
+/** @ngInject */
+export function autofillEventFix($compile) {
+  return {
+    link: ($scope: any, elem: any) => {
+      const input = elem[0];
```

Comment:
What do you think about returning from `link` if user is on `iOS`? This would save us some innecessary change events you have mentioned.